src/gfx/context.cpp

```diff
@@ -153,7 +153,10 @@ struct ContextMainOutput {
 
   void resizeSwapchain(WGPUDevice device, WGPUAdapter adapter, const int2 &newSize) {
     WGPUTextureFormat preferredFormat = wgpuSurfaceGetPreferredFormat(wgpuWindowSurface, adapter);
-    preferredFormat = WGPUTextureFormat_BGRA8Unorm;
+
+    // Force the backbuffer to srgb format so we don't have to convert manually in shader
+    preferredFormat = WGPUTextureFormat_BGRA8UnormSrgb;
```

Comment:
That's a hard choice.
We all good with this @guusw @inabe-a @Kryptos-FR ?
Afaik conversion in the shader leaves everything very flexible and the cost is tiny. But this was the case mostly with bgfx supporting opengl and dx9... with webgpu we can just forget that and use Srgb.